content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -95,14 +95,30 @@ using the online documentation, the glossary entry displays a tooltip.
 
 The raw data for glossary terms is stored at [https://github.com/kubernetes/website/tree/master/content/en/docs/reference/glossary](https://github.com/kubernetes/website/tree/master/content/en/docs/reference/glossary), with a content file for each glossary term.
 
-### Glossary Demo
+### Glossary demo
 
 For example, the following include within the markdown renders to {{< glossary_tooltip text="cluster" term_id="cluster" >}} with a tooltip:
 
-```liquid
+```
 {{</* glossary_tooltip text="cluster" term_id="cluster" */>}}
 ```
 
+As well as tooltips, you can reuse the definitions from the glossary in page content.  
+Here's a short glossary definition:
+
+```
+{{</* glossary_definition prepend="A cluster is" term_id="cluster" length="short" */>}}
+```
+which renders as:
+{{< glossary_definition prepend="A cluster is" term_id="cluster" length="short" >}}</i>
+
+You can also include a full definition:
+```
```

Comment:
nit: You may want to indicate that there are two acceptable strings for generating the full definition:
`all` and `long`. Otherwise, if no length attribute is set, the definition generated is the `short` definition.

Reply:
If no length attribute is set, this revised shortcode fails to render. That's my plan.

Reply:
TBH, I don't like the 'prepend' parameter. For example, converting this to the rendered output is always painful when I'm creating or reviewing localized contents. The reason is that sentences are structured in different ways in different languages. Maybe some Japanese localization contributors can chime in as well?

I'd prefer leave the shortcode as simple as it can be.

Reply:
In English, `prepend` lets you incorporate a glossary definition into running text. In which localizations is this difficult, and what changes could help there?

Also by forcing users to select a length it avoids people accidentally mixing it up with `{{< glossary_tooltip … >}}` - adding this check let me spot a case where that had happened.